docker-compose.yml

```diff
@@ -1,22 +1,25 @@
-version: '3'
+version: '3.5'
+
 services:
-    ganache:
-        image: '0xorg/ganache-cli:6.5.10'
-        ports:
-            - '8545:8545'
-        environment:
-            VERSION: 8.0.1
-            CHAIN_ID: 1337
-            NETWORK_ID: 1337
-    postgres:
-        image: postgres:9.6
-        environment:
-            - POSTGRES_USER=api
-            - POSTGRES_PASSWORD=api
-            - POSTGRES_DB=api
-        # persist the postgres data to disk so we don't lose it
-        # on rebuilds.
-        volumes:
-            - ./postgres:/var/lib/postgresql/data
-        ports:
-            - '5432:5432'
\ No newline at end of file
+  postgres:
+    container_name: postgres_container
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
Is there a need to push this to github?

Reply:
I will add this information to .env file.